src/math/Vector2.js

```diff
@@ -370,6 +370,18 @@ THREE.Vector2.prototype = {
 
 	},
 
+	angle: function () {
+
+		// computes the angle in radians with respect to the positive x-axis
+
+		var angle = Math.atan2( this.y, this.x );
+
+		if ( angle < 0 ) angle += 2 * Math.PI;
```

Comment:
I see I see. It was in case someone expected the `- PI` to `+PI` output. Looks good then!